src/main/java/de/leonhard/storage/Config.java

```diff
@@ -4,35 +4,38 @@
 import de.leonhard.storage.internal.settings.ConfigSettings;
 import de.leonhard.storage.internal.settings.DataType;
 import de.leonhard.storage.internal.settings.ReloadSettings;
+import lombok.NonNull;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Why use '@NotNull' when you can use the Lombok version '@NonNull' I mean you use both but everything newly added is the jet brains one.